### DISCUSSION TITLE
Confirmando prazo

### DISCUSSION THREAD

**Discussion by mateusasferreira** [Nexo-Tec/time#15] [Geral]
Boa tarde, tudo bem?
Acabei de ver que minha pergunta, que era sobre o prazo, já foi respondida. Porém, gostaria de comunicar que fiquei sabendo da vaga ontem somente e ficaria feliz se contassem com minha candidatura mesmo perto do prazo final. Digo isso porque tive problemas com a ativação da minha conta AWS (não recebi o código de verificação por SMS, entrei em contato com o atendimento hoje e eles me disseram que resolveriam em até 48h). Enfim, em breve estarei enviando a candidatura e também reportando como resolvi esse pequeno problema hehe. Abraços!

**Comment by ibrahimcesar** [comment]
Não se preocupe que continuaremos aceitando candidaturas até encontrar as pessoas para o time! Pode tomar o seu tempo!

**Reply by mateusasferreira** [reply]
Perfeito, valeu!

**Comment by NixLoar** [comment]
Olá, pessoal!! Boa noite :) 

Em vez de abrir um post novo, vou aproveitar esse já que é sobre o mesmo assunto haha

Tomei conhecimento da vaga a 2 dias atrás e super me interessei no projeto. Entretanto, infelizmente ainda não tive tempo para desenvolver algo e o prazo está se esgotando. Existe alguma perspectiva do processo seletivo ser estendido por mais um tempinho?

**Reply by ibrahimcesar** [reply]
Olá @Caiolps ,

Desculpe pela demora em resposta. Segunda-feira tínhamos como marco completar nossas entrevistas e iniciar os convites para aquelas pessoas que selecionarmos. O que não significa que todas irão aceitar. Se já está engajado em algum desafio ou deseja nos mostrar algo, pode submeter a qualquer momento – só não podemos garantir que já seja avaliado nesse processo específico que fizemos ao longo de Maio. Também estamos pensando em formas de manter esse tipo de submissão ao longo do tempo. Mas ao mesmo tempo também não pretendo criar expectativas, tivemos excelentes submissões e como segunda teremos as últimas entrevistas marcadas, já estamos com uma visão mais ou menos solida do que seguir no início de junho. Mas caso esteja motivado, iremos avaliar de qualquer forma.

Obrigado pelo interesse!
